lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -50,6 +50,8 @@ public sealed class REEFFileNames
         private const string CLR_DRIVER_CONFIGURATION_NAME = "clrdriver.conf";
         private const string BRIDGE_DLL_NAME = "Org.Apache.REEF.Bridge.dll";
 
+        public const string CLR_BRIDGE_CONFIGURATION_NAME = "clrBridge.config";
```

Comment:
The idea for `REEFFileNames` was to have it injected in all places where it is used. That allows us to evolve it over time. Hence, a `public const` doesn't mesh with that idea.